app/views/mod/communication.scala.html

```diff
@@ -71,7 +71,9 @@
     case (pov, chat) => {
     <div class="game">
       <a class="title" href="@routes.Round.player(pov.fullId)">
-        vs @usernameOrAnon(pov.opponent.userId) @momentFromNow(pov.game.movedAt)
+        @if(pov.game.fromFriend) {
+          <strong>[F]</strong>
```

Comment:
obscure. Please add a title attribute.

Reply:
sorry I don't understand. what do you mean?